File: domain/aplicativo/usuario.py
import musicas.bandas_favoritas as bandas_favs_ent
import musicas.playlists as playlists_ent

class Usuario:
    
    def __init__(self, nome, sobrenome, cartao_credito):
        
        # dados usuario
        self.nome = nome
        self.sobrenome = sobrenome
        self.cartao_credito = cartao_credito
        
        # assinatura
        self.assinaturas = []
        
        # playlists e bandas
        self.playlists = playlists_ent.Playlists(self)
        self.bandas_favoritas = bandas_favs_ent.BandasFavoritas(self)
    
    
    def criar_usuario(cls, nome, sobrenome, cartao_credito):
        usuario = Usuario.criar_usuario(nome, sobrenome, cartao_credito)
        return usuario

File: MusicApp/MusicApp.Domain/Usuario/Aggregates/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MusicApp.Domain.Aplicativo.Aggregates;


namespace MusicApp.Domain.Usuario.Aggregates
{
    public class Usuario
    {

        // Info
        public Guid Id { get; set; }
        public String Nome { get; set; }

        // Pagamento
        public List<CartaoCredito> Cartoes { get; set; }
        public List<Assinatura> Assinaturas { get; set; }

        // App
        public List<Playlist> Playlists { get; set; }
        public List<Banda> BandasFavoritas { get; set; }

        // Construtor
        public Usuario() 
        {
            this.Playlists = new List<Playlist>();
            this.Playlists.Add(new Playlist() { Nome = "Favoritas" });

            this.BandasFavoritas = new List<Banda>();
            this.Assinaturas = new List<Assinatura>();
            this.Cartoes = new List<CartaoCredito>();
        }


        public static Usuario CriarUsuario() { return new Usuario(); }

        public void AdicionarCartaoCredito(CartaoCredito cartao)
        {
            this.Cartoes.Add(cartao);
        }

        public void AssinarPlano(Plano plano, CartaoCredito cartao) { }

        public void CriarPlaylist(string nome) { }

        public void FavoritarBanda(string nome) { }

    }
}
